frontend/src/helpers/date/get-formatted-date/get-formatted-date.helper.ts

```diff
@@ -0,0 +1,15 @@
+import { format } from 'date-fns';
+
+const getFormattedDate = (
```

Comment:
Let's use `getFormattedDate` helper from shared folder (you may need to enhance that helper to provide your date formatting template)